calyx/src/pass_manager.rs

```diff
@@ -176,6 +176,27 @@ impl PassManager {
 
         Ok(())
     }
+
+    /// Executes a given "plan" constructed with logging disabled
+    pub fn execute_plan_without_log(
```

Comment:
I don't know if this is a good idea, but one option would be to use [a wasm-specific logger](https://crates.io/crates/wasm-logger), sort of analogously to the wasm-specific panic hook.

Reply:
Thanks for the reference! I'm trying to see if there is a shim for the Instant::now() implementation somewhere that I can use. If not, this might be the way to go.